view/frontend/templates/menu/node_type/custom_url.phtml

```diff
@@ -13,9 +14,16 @@ if ($block->getIsViewAllLink()) {
         $dataAttribute = ' data-menu="menu-' . $block->getId() . '"';
     }
 }
+
+if($block->getTarget()) {
+    $target = 'target="__blank"';
```

Comment:
Please consirer also add `rel="noopener"` https://mathiasbynens.github.io/rel-noopener/ .

Reply:
@talalus that's a really good advice, let's do it

Reply:
done